src/operators.rs

```diff
@@ -285,6 +301,46 @@ impl Initializer<FenderReference> for FenderInitializer {
                 }
                 FenderValue::String(collected.into()).into()
             }
+            FenderInitializer::Struct(id) => {
+                match ctx.context.struct_table.type_list().get(*id).cloned() {
+                    Some(struct_type) => {
```

Comment:
This match arm is large enough that I think it constitutes being its own function